include/netutils/webclient.h

```diff
@@ -49,6 +49,7 @@
 #include <nuttx/config.h>
 
 #include <stdbool.h>
+#include <stdint.h>
```

Comment:
@yamt  BTW I noticed that `stdint` type is used only in `uint16_t tunnel_target_port;`, so maybe it is reasonable to change it to `unsigned int tunnel_target_port;` and do not include `stdint.h`?

Reply:
Anyway that is minor and current type is used to match `struct wget_target_s`.

Reply:
port number is 16 bit.
using a wider type makes it more difficult to understand, both for human beings and tools.